dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/program/variable/ProgramVariableItem.java

```diff
@@ -49,6 +49,7 @@
         .put( V_ANALYTICS_PERIOD_START, new vAnalyticsPeriodStart() )
         .put( V_CREATION_DATE, new vCreationDate() )
         .put( V_CURRENT_DATE, new vCurrentDate() )
+       // .put( V_COMPLETED_DATE, new vCompletedDate() )
```

Comment:
I'm guessing this is commented out because https://github.com/dhis2/dhis2-antlr-expression-parser/pull/7 had not yet been merged. But perhaps it could be commented back in now, since the other PR has now been merged?